### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by johng42**
issue 260 https://github.com/tableau/TabPy/issues/260

### CODE REVIEW

File: tabpy-server/tabpy_server/app/util.py [hunk 1]
```diff
@@ -36,8 +36,8 @@ def validate_cert(cert_file_path):
                           not_before), RuntimeError)
     if now > not_after:
         log_and_raise(https_error +
-                      f'The certificate provided expired on {not_after}.',
-                      RuntimeError)
+                      'The certificate provided expired on {}.'.format(
```
Comment (jnegara) [inline tabpy-server/tabpy_server/app/util.py]: Why is this change necessary?

Comment (johng42) [reply tabpy-server/tabpy_server/app/util.py]: you get a syntax error when the code is run

Comment (johng42) [reply tabpy-server/tabpy_server/app/util.py]: only in the sense that the logging syntax for expired now matches the syntax for not_before
